test/main.js

```diff
@@ -104,17 +113,6 @@ describe('parse-conf', () => {
   });
 });
 
```

Comment:
A test for this may be overkill, but can we confirm that we get a 404 for requests to `/robots.txt`?